modules/core/src/test/java/org/apache/ignite/internal/metric/JmxExporterSpiTest.java

```diff
@@ -730,6 +735,105 @@ public void testScanQuery() throws Exception {
         }
     }
 
+    /**
+     *
```

Comment:
nit: Let's write this comment in one line.

Reply:
done